src/gtimelog/timelog.py

```diff
@@ -249,7 +249,13 @@ def split_category(entry):
         Return a tuple (category, task).
         """
         if ': ' in entry:
-            return tuple(entry.split(': ', 1))
+            cat, task = entry.split(': ', 1)
+            t, _, c = cat.partition('@')
+            if c and t:
+                # if category string can be further partitioned into non empty strings by '@'
+                # use 1st as task and 2nd as category
+                cat, task = c, t
+            return cat, task
```

Comment:
Given #181 you'll probably want to add a few .strip()s here and thee.

Reply:
Probably Yes. I wasn’t sure if you accept #181.